src/ResourceManager/Compute/Commands.Compute/AvailabilitySets/NewAzureAvailabilitySetCommand.cs

```diff
@@ -46,7 +47,7 @@ public class NewAzureAvailabilitySetCommand : AvailabilitySetBaseCmdlet
             Mandatory = true,
             Position = 2,
             ValueFromPipelineByPropertyName = true,
-            HelpMessage = "The location.")]
+            HelpMessage = "The location."), LocationCompleter(new string[] { "Microsoft.Compute/availabilitySets" })]
```

Comment:
Move LocationCompleter() to [ValidateNotNullOrEmpty] like NewAzureVMCommand.cs

Reply:
I moved all of the LocationCompleter attributes out to a new set of brackets for uniformity - let me know if this looks good to you.